ui/components/DryRunComponent.js

```diff
@@ -0,0 +1,326 @@
+import React from "react";
```

Comment:
Please organize the components into respective folders. Create new folders as needed and move components around as you see fit.

Reply:
The Patterns/Applications components were in root , so put it there also . i will create a new folder for this .

Reply:
Thank you much. Let's break those old bad habits.